main.go

```diff
@@ -26,6 +27,7 @@ var (
 	isDebug       = flag.Bool("debug", false, "Output verbose debug information")
 	logFormat     = flag.String("log-format", "txt", "Log format, valid options are txt and json")
 	showVersion   = flag.Bool("version", false, "Show version information and exit")
+	useCfBinding  = flag.Bool("use-cf-binding", false, "Use Cloud foundry service bindings")
```

Comment:
`use-cf-binding` --> `use-cf-bindings`
`Cloud foundry` -> `Cloud Foundry`

can you please also add an entry to the README for this option?

Reply:
Done

Reply:
Thx. This is still missing the entry in the README though.